tests/test_loaders.py

```diff
@@ -0,0 +1,8 @@
+from aiidalab_widgets_base.utils.loaders import load_css_stylesheet
+
+
+def test_load_css_stylesheet():
+    """Test `load_css_stylesheet` function."""
+    package = "aiidalab_widgets_base.static.styles"
+    load_css_stylesheet(package=package, filename="global.css")
+    load_css_stylesheet(package=package)
```

Comment:
The test here will not work because it is not running inside a notebook. To test whether the CSS works, you need to add a test in `test_notebooks/test_notebooks.py.`

Reply:
Are you saying to add a test to test_notebooks.py, or to move this test to test_notebooks.py? Also, could you explain why exactly it doesn't work?

Reply:
Yes, adding a new test in the test_notebooks.py, and check if the CSS is indeed loaded in the selenium_driver.

> Also, could you explain why exactly it doesn't work?

the `test.py` file is not running in the notebook, thus the `display(Javascript...` code will not run. You can add some wrong code inside it, and the test will not throw error.

Reply:
For the test in selenium, for example, you create a vbox, and attach a class, then in your testing CSS  file, you change its color, and in selenium_driver, you try to find the vbox, and verify the color.

Reply:
The test here is fine, I think but I agree it would be nice to have *an additional` test in `test_notebooks` that runs an actual browser via selenium that would prove that the CSS is indeed loaded and applied to a given element (I don't actually know if selenium allows that but I would expect so).

Reply:
Will add. Thanks

Reply:
@superstar54 I'm not familiar with selenium. I see that you have to set a driver which wraps on an existing notebook. I then do `driver.find_element`, but does that only check THAT notebook? If so, which notebook do I use?

Reply:
I think you can create a new notebook (e.g., `test_css.ipynb`), and in one cell you add
```python
label = ipw.Label("Testing")
label.add_class("for-testing-purposes")
display(label)
```

in your test
```python
def test_load_css_stylesheet(selenium_driver):
    """Test `load_css_stylesheet` function."""
    driver = selenium_driver("notebooks/test_css.ipynb")
    driver.find_element(By.CLASS_NAME, "for-testing-purposes")
```
you may investigate how to check the style of the element, I don't know the how to do this exactly.

Reply:
Sure, if you guys don't mind another notebook just for testing. Though maybe just call it `test.ipynb`? You never know what other tests might need a dedicated notebook🤷🏻‍♂️

Reply:
I feel like there is a bit of misunderstanding here. The purpose of the notebook test is not to test the load_css_stylesheet on its own. It should be mainly test that the CSS style is actually applied. 

If you added the CSS code for the DictViewer, you could just extend the existing DictViewer test. As @superstar54 said, you'll need to investigate if selenium allows you to actually test CSS properties. If not, then adding a notebook test does not give us much.

The test itself shouldn't be using `load_css_stylesheet`, it should be loaded implicitly when awb is imported.

Reply:
Take a look at the notebook test I pushed. It does exactly that - it checks that a property of a class is present on a widget. However, it does use `load_css_stylesheet` in the test. This would be redundant if awb is imported, but is it imported anywhere? I only imported it in the test to use `load_css_stylesheet`. If I remove the call, where would awb be called?

Reply:
Yes, in the test notebook, you need at least 
```python
import aiidalab_widgets_base
```

Reply:
Why?

Reply:
I mean you only need to import `aiidalab_widgets_base`, since it will call `load_css_stylesheet` in the `__init__.py` file.

Reply:
I've loaded `aiidalab_widgets_base` in the `test.ipynb` notebook with a comment regarding its use, as it is not actually used explicitly.

Reply:
Okay, notebook test passes 👍